core/src/main/java/com/linecorp/armeria/client/limit/ConcurrencyLimitingClient.java

```diff
@@ -62,6 +63,26 @@
         return newDecorator(limit);
     }
 
+    /**
+     * Creates a new {@link HttpClient} decorator that limits the concurrent number of active HTTP requests.
+     */
+    public static Function<? super HttpClient, ConcurrencyLimitingClient> newDecorator(
+            SettableLimit maxConcurrency) {
+        final ConcurrencyLimit limit = ConcurrencyLimit.of(maxConcurrency);
+        return newDecorator(limit);
+    }
+
+    /**
+     * Creates a new {@link HttpClient} decorator that limits the concurrent number of active HTTP requests.
+     */
+    public static Function<? super HttpClient, ConcurrencyLimitingClient> newDecorator(
+            SettableLimit maxConcurrency, long timeout, TimeUnit unit) {
```

Comment:
This is not your fault.
But we no longer take `long timeout, TimeUnit unit` parameter together anymore. Instead, we take `long timeoutMillis` or `Duration timeout`.
So, how about just removing this method and deprecating the method above
`... newDecorator(int maxConcurrency, long timeout, TimeUnit unit) {...}`?
We also need to add `@deprecate Use .....` to the method Javadoc.